lib/internal/bootstrap_node.js

```diff
@@ -420,7 +427,7 @@
     const module = new Module(name);
     module.filename = path.join(cwd, name);
     module.paths = Module._nodeModulePaths(cwd);
-    const body = process._eval;
+    const body = wrapForBreakOnFirstLine(process._eval);
     const script = `global.__filename = ${JSON.stringify(name)};\n` +
```

Comment:
would it work better if you applied `wrapForBreakOnFirstLine` to `script`, not to `body`?